patterns/nvidia-gpu-efa/README.md

```diff
@@ -82,84 +88,178 @@ See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started
+    It is normal for the launcher pod to restart a few times until the worker pods are fully running.
+
+    ```sh
+    watch kubectl get pods
+    ```
+
+    ```log
+    NAME                           READY   STATUS             RESTARTS      AGE
+    efa-info-test-launcher-wm8pm   0/1     CrashLoopBackOff   1 (16s ago)   19s
+    efa-info-test-worker-0         1/1     Running            0             19s
+    efa-info-test-worker-1         1/1     Running            0             19s
+    ```
+
+    ```log
+    NAME                           READY   STATUS    RESTARTS      AGE
+    efa-info-test-launcher-wm8pm   1/1     Running   2 (18s ago)   21s
+    efa-info-test-worker-0         1/1     Running   0             21s
+    efa-info-test-worker-1         1/1     Running   0             21s
+    ```
+
+    ```log
+    NAME                           READY   STATUS      RESTARTS   AGE
+    efa-info-test-launcher-wm8pm   0/1     Completed   2          5m20s
     ```
 
-    Once the test launcher pod enters status `Running` or `Completed`, see the test logs using the command below:
+    Once the test launcher pod enters status `Running` or `Completed`, 
+    see the test logs using the command below:
 
     ```sh
     kubectl logs -f $(kubectl get pods | grep launcher | cut -d ' ' -f 1)
     ```
 
-    ```text
-    Warning: Permanently added 'efa-info-test-worker-1.efa-info-test-worker.default.svc,10.11.13.224' (ECDSA) to the list of known hosts.
-    Warning: Permanently added 'efa-info-test-worker-0.efa-info-test-worker.default.svc,10.11.4.63' (ECDSA) to the list of known hosts.
+    ```log
+    Warning: Permanently added 'efa-info-test-worker-1.efa-info-test.default.svc' (ED25519) to the list of known hosts.
+    Warning: Permanently added 'efa-info-test-worker-0.efa-info-test.default.svc' (ED25519) to the list of known hosts.
     [1,1]<stdout>:provider: efa
     [1,1]<stdout>:    fabric: efa
-    [1,1]<stdout>:    domain: rdmap197s0-rdm
-    [1,1]<stdout>:    version: 116.10
+    [1,1]<stdout>:    domain: rdmap79s0-rdm
+    [1,1]<stdout>:    version: 120.10
     [1,1]<stdout>:    type: FI_EP_RDM
     [1,1]<stdout>:    protocol: FI_PROTO_EFA
+    
+    ...
+    
     [1,0]<stdout>:provider: efa
     [1,0]<stdout>:    fabric: efa
-    [1,0]<stdout>:    domain: rdmap197s0-rdm
-    [1,0]<stdout>:    version: 116.10
+    [1,0]<stdout>:    domain: rdmap201s0-rdm
+    [1,0]<stdout>:    version: 120.10
     [1,0]<stdout>:    type: FI_EP_RDM
     [1,0]<stdout>:    protocol: FI_PROTO_EFA
     ```
 
-4. EFA NCCL test
+    Finally, remove the job:
+    
+    ```sh
+    kubectl delete -f ./efa-info-test.yaml
+    ```
 
-    To run the EFA NCCL test please execute the following kubectl command:
+4. EFA NCCL test
 
+    The EFA NCCL test is used to measure network bandwidth by running the `/opt/nccl-tests/build/all_reduce_perf` utility. 
+    Please edit the script below and modify the environment variables at the beginning of the script as needed. 
```

Comment:
likewise, make it static - here we are using a `p5.48xlarge` so those values are known